store/snap_remote_repo.go

```diff
@@ -292,7 +422,8 @@ func (s *SnapUbuntuStoreRepository) Snap(name, channel string, auther Authentica
 	// store this is not a concern right now.
 	for _, pkg := range searchData.Payload.Packages {
 		if pkg.Name == name {
-			return infoFromRemote(pkg), nil
+			purchases, _ := s.getPurchases(pkg.Name+"."+pkg.Developer, auther)
```

Comment:
Whole code needs to be adapted to current conventions (no Developer, snap rather than package, etc).